nuxt/api/albums.ts

```diff
@@ -0,0 +1,72 @@
+import { NuxtAxiosInstance } from '@nuxtjs/axios';
+import {
+  PaginationOptions,
+  PersistedEntity,
+  TimestampedEntity,
+  PaginatedResponse,
+  createParams,
+  usePagination,
+  useIncludes,
+} from '@/api/common';
+import { Reciter } from '@/api/reciters';
+
+/*
+ * Entity Definitions
+ */
+export interface Album extends PersistedEntity, TimestampedEntity {
+  reciter?: Reciter;
+  reciterId: string;
+  title: string;
+  year: string;
+  artwork: string;
```

Comment:
We're missing `tracks` as a property here, which can be present if you use the `tracks` include.

```suggestion
  artwork: string;
  tracks: EntityCollection<Track>;
```

You can define the `EntityCollection` interface in `api/common.ts` as follows:

```ts
export interface EntityCollection<T> {
  data: Array<T>;
}
```
```